DEVELOPMENT.md

```diff
@@ -11,7 +10,6 @@ git clone https://github.com/cryptoadvance/specter-desktop.git
 cd specter-desktop
 virtualenv --python=python3 .env
 source .env/bin/activate
-pip3 install -r requirements.txt
```

Comment:
Not sure why you remove it here. IMHO for Development, you need both?
You want to be able to run the tests locally.

Reply:
It installs all the requirements with `pip3 install -e .` anyways